src/Confluent.Kafka/Config_gen.cs

```diff
@@ -1,4 +1,4 @@
-// *** Auto-generated from librdkafka v1.0.0 *** - do not modify manually.
+// *** Auto-generated from librdkafka master *** - do not modify manually.
```

Comment:
nit: Would prefer a proper tag / sha1 here.

Reply:
v. good idea re: sha1. ideally we want the proper tag of the actual librdkafka release, but that doesn't exist yet so it would result in just an extra step that isn't worth it just for a comment.